pyartcd/pyartcd/pipelines/ocp4.py

```diff
@@ -677,6 +677,13 @@ async def _build_images(self):
             cmd.extend(['images:streams', 'mirror'])
             await exectools.cmd_assert_async(cmd)
 
+        # Kick off SAST scans for builds that succeeded using the NVR
+        successful_build_nvrs = []
+        for build in success_map.values():  # since full_record=True
+            successful_build_nvrs.append(build["nvrs"])
```

Comment:
```suggestion
            successful_build_nvrs = [build["nvrs"] for build in success_map.values()]
```

Reply:
added in https://github.com/openshift-eng/aos-cd-jobs/pull/3903/commits/fc4b81b679e84014c62aee38a4576bc32ad71af1